linkerd/protocol/http/src/main/scala/io/buoyant/linkerd/protocol/HttpConfig.scala

```diff
@@ -37,6 +37,21 @@ class HttpInitializer extends ProtocolInitializer.Simple {
       .configured(RoutingFactory.DstPrefix(Path.Utf8(name)))
   }
 
+  /**
+   * Apply the router's codec coonfiguration applies to the server.
```

Comment:
typo: "coonfiguration" (and I think the "applies" may be extraneous, unless I misunderstand its intent?)